src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -2126,6 +2126,17 @@ protected function get_unique_field( $for = null ) {
 	 */
 	public function finalize() {
 		$this->update_meta( 'finalized', true );
+
+		/**
+		 * Fires after a record has been finalized and right before it starts importing.
+		 *
+		 * @since TBD
+		 *
+		 * @param int $id     The Record post ID
```

Comment:
100% optional nitpick: since we're aligning param descriptions, aligning $vars with one another might add some further polish.